pkg/kudoctl/cmd/init.go

```diff
@@ -98,6 +99,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.BoolVar(&i.crdOnly, "crd-only", false, "Add only KUDO CRDs to your cluster")
 	f.BoolVarP(&i.wait, "wait", "w", false, "Block until KUDO manager is running and ready to receive requests")
 	f.Int64Var(&i.timeout, "wait-timeout", 300, "Wait timeout to be used")
+	f.BoolVar(&i.enableValidation, "enable-validation", false, "When set to true, validation webhook will be deployed alongside KUDO controller")
```

Comment:
as we discussed... I think this should be more clear and likely more granular... perhaps `--web-hooks=InstanceValidator`
with a comma separated list... and perhaps `--web-hooks=All` for convince.
Not a blocker for me.. just think we will have to address this at some point.